package.json

```diff
@@ -25,7 +22,7 @@
     "build:schematics:collection": "cpy --parents --cwd='src/schematics/src/' '**/*.json' ../../../dist/clr-angular/schematics",
     "build:schematics": "npm-run-all build:schematics:ts build:schematics:collection",
     "build:golden": "ts-api-guardian --out dist/golden/clr-angular.d.ts --stripExportPattern 'FocusTrapTracker' dist/clr-angular/index.d.ts",
-    "build": "npm-run-all clean build:ui build:icons build:angular build:base build:schematics build:golden",
+    "build": "npm-run-all clean build:ui build:icons build:angular base:build build:schematics build:golden",
```

Comment:
Why do we need to build base as part of the full build at the current moment?
Just curious.

Reply:
I know that the dev app has been using base for the past couple of months.

Reply:
I would like it to be so the dev app can continue to be integrated and tested. But if it does start to cause problems I will disable it until we get further along.

Reply:
Didn't know it's in the dev app. Thanks